### PR TITLE
The Forkening:  Change the name EtherDelta to ForkDelta in text and references.

### PR DESCRIPTION
**Pull request by JonathonDunford**
@freeatnet Minify plz

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-EtherDelta's API allows you to query orders, trades, deposits, and withdrawals. Since EtherDelta is a smart contract, it is also possible to get trades, deposits, and withdrawals from the Ethereum API (through an Ethereum node or Etherscan API). EtherDelta's orderbook is an off-chain mechanism (a list of cryptographically signed orders) that you can only access through this API. The API allows you to see existing orders or place a new one.
+ForkDelta's API allows you to query orders, trades, deposits, and withdrawals. Since ForkDelta is a smart contract, it is also possible to get trades, deposits, and withdrawals from the Ethereum API (through an Ethereum node or Etherscan API). ForkDelta's orderbook is an off-chain mechanism (a list of cryptographically signed orders) that you can only access through this API. The API allows you to see existing orders or place a new one.
 
 ## Server
 
-EtherDelta has one API endpoint. It is a Websocket API.
+ForkDelta has one API endpoint. It is a Websocket API.
 
- * `https://socket.etherdelta.com`
+ * `https://api.ForkDelta.com`
```
Comment (Skhtor) [inline docs/API.md]: Best leave the URL in lower case?

File: gntw.html [hunk 2]
```diff
@@ -216,9 +216,6 @@ <h3>Convert GNTW &#8212;&gt; GNT</h3>
       ga('send', 'pageview');
     </script>
     <script>
-      ((window.gitter = {}).chat = {}).options = {
```
Comment (Skhtor) [inline gntw.html]: Just out of curiosity, do we know what these lines did?

File: docs/API.md (hunk 1 above)
Comment (freeatnet) [reply docs/API.md]: +1

File: gntw.html (hunk 2 above)
Comment (freeatnet) [reply gntw.html]: This was ED's gitter-based support chat. I guess that's how you initialize its widget.

File: gntw.html [hunk 3]
```diff
@@ -216,9 +216,6 @@ <h3>Convert GNTW &#8212;&gt; GNT</h3>
       ga('send', 'pageview');
     </script>
     <script>
```
Comment (freeatnet) [inline gntw.html]: We should remove empty `<script>` blocks.

File: docs/API.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
-# EtherDelta API
+# ForkDelta API
 
-EtherDelta's API allows you to query orders, trades, deposits, and withdrawals. Since EtherDelta is a smart contract, it is also possible to get trades, deposits, and withdrawals from the Ethereum API (through an Ethereum node or Etherscan API). EtherDelta's orderbook is an off-chain mechanism (a list of cryptographically signed orders) that you can only access through this API. The API allows you to see existing orders or place a new one.
+ForkDelta's API allows you to query orders, trades, deposits, and withdrawals. Since ForkDelta is a smart contract, it is also possible to get trades, deposits, and withdrawals from the Ethereum API (through an Ethereum node or Etherscan API). ForkDelta's orderbook is an off-chain mechanism (a list of cryptographically signed orders) that you can only access through this API. The API allows you to see existing orders or place a new one.
```
Comment (freeatnet) [inline docs/API.md]: > ForkDelta's API
✔️ 

> ForkDelta is a smart contract

ForkDelta isn't a smart contract and currently does not operate one on its own.

File: docs/API.md [hunk 5]
```diff
@@ -131,7 +131,7 @@ This allows you to post an order. `order` should be a JSON object with the follo
 * `tokenGive`: the token you want to give (use the zero address, `0x0000000000000000000000000000000000000000` for ETH)
 * `amountGet`: the amount you want to get (in wei or the base unit of the token)
 * `tokenGet`: the token you want to get (use the zero address, `0x0000000000000000000000000000000000000000` for ETH)
-* `contractAddr`: the EtherDelta smart contract address
+* `contractAddr`: the ForkDelta smart contract address
```
Comment (freeatnet) [inline docs/API.md]: Perhaps use "the current smart contract address"?

File: index.html [hunk 6]
```diff
@@ -121,25 +121,26 @@
...
+                        This interface is a fork of <a href="https://etherdelta.com">EtherDelta</a> with a ton of improvements including a much faster order processing system.
+                        It interacts with ForkDelta's revamped API and EtherDelta's original contract.
                       </p>
                       <p style="color: #f00; font-weight: bold;">How to Start Trading</p>
                       <p style="font-size: 12px;">
-                        <strong>Use the Deposit form (upper left) to put your funds (ETH and tokens) on the exchange.</strong> Do not send your tokens directly to the smart contract, or they will be lost and unrecoverable. ForkDelta cannot help you with funds incorrectly deposited to EtherDelta contract.
+                        <strong>Use the Deposit form (upper left) to put your funds (ETH and tokens) on the exchange.</strong> Do not send your tokens directly to the smart contract, or they will be lost and unrecoverable. ForkDelta cannot help you with funds incorrectly deposited to ForkDelta contract.
```
Comment (freeatnet) [inline index.html]: "the trading smart contract"?

File: js/main-unminified.js [hunk 7]
```diff
@@ -131664,50 +131664,50 @@
...
                             ko:
-                                '계정 드롭다운(우측 상단)에 표시 중인 EtherDelta와 연결된 당신의 이더리움 개인지갑의 잔고입니다.',
+                                '계정 드롭다운(우측 상단)에 표시 중인 ForkDelta와 연결된 당신의 이더리움 개인지갑의 잔고입니다.',
                         },
-                        balance_etherdelta_tooltip: {
+                        balance_ForkDelta_tooltip: {
                             en:
-                                'This is the balance you have deposited from your personal Ethereum wallet to the EtherDelta smart contract.',
-                            cn: '这是您从您的个人Ethereum钱包存储到EtherDelta智能合同的余额。',
+                                'This is the balance you have deposited from your personal Ethereum wallet to the ForkDelta smart contract.',
```
Comment (freeatnet) [inline js/main-unminified.js]: Ditto.

**Comment by JonathonDunford** [comment]
@freeatnet Updated.

**Pull request merged**